R/Sound_DMR.R

```diff
@@ -566,21 +566,21 @@ run_binomial <- function(LM, i = int, formula,
 #' @inheritParams run_binomial
 #' @param individual_name_z (str) - *optional* the name of the z column when running
 #' individual DMR analysis
-#' @return Output_Frame (df) - updated with the summary data from the model
+#' @return Input_frame (df) - updated with the summary data from the model
```

Comment:
super nit but wondering if we should maintain capitalization consistency? So either like "Input_Frame" or "input_frame" or "InputFrame", I don't know if there's a preferred convention in R

Reply:
I agree 100%! Again this goes along with the formatting I was talking about. Right now we can clearly distinguish between the 3 of us who wrote the functions because of the style, format and other nit picky stuff.